core/src/main/java/com/linecorp/armeria/common/NonWrappingRequestContext.java

```diff
@@ -223,6 +223,15 @@ public final Iterator<Entry<AttributeKey<?>, Object>> ownAttrs() {
         return attrs.ownAttrs();
     }
 
+    /**
+     * Returns the {@link Attributes} which stores the pairs of a {@link AttributeKey} and a object set via
```

Comment:
Since it is possible that `ConcurrentAttributes` is returned as well

```suggestion
     * Returns the {@link AttributeGetters} which stores the pairs of a {@link AttributeKey} and a object set via
```